include/galsim/hsm/PSFCorr.h

```diff
@@ -122,6 +122,10 @@ namespace hsm {
      *                           KSB method of PSF correction.  Warning: deviating from default
      *                           value of 2 results in code running more slowly, and results have
      *                           not been significantly tested.
+     * @param epsilon            Accuracy (in x0, y0, and sigma as a fraction of sigma) when
```

Comment:
I looked into this a bit more, and I convinced myself that it is indeed correct that dx and dy are in fractions of sigma, so I updated the doc string for this to be: "Accuracy (in x0, y0, and sigma, each as a fraction of sigma) when calculating adaptive moments. ..." as you suggested, Josh.